tests/test_wallet.py

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2020, hukkinj1 (licensed under the MIT License)
 # Modifications Copyright (c) 2020, Foris Limited (licensed under the Apache License, Version 2.0)
 
-from chainlibpy import Wallet
+from chainlibpy.wallet import Wallet
 
 
 def test_generate_wallet():
-    seed = "burst negative solar evoke traffic yard lizard next series foster seminar enter wrist captain bulb trap giggle country sword season shoot boy bargain deal"
+    seed = "burst negative solar evoke traffic yard lizard next series foster seminar enter wrist captain bulb trap giggle country sword season shoot boy bargain deal"  # noqa 501
```

Comment:
what does it mean "noqa 501"?

Reply:
Adding `# noqa 501` at the end of line tells `flake8` to ignore `E501 line too long` error.
https://stackoverflow.com/a/47876831/17495841